<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddEntry</class>
 <widget class="QDialog" name="AddEntry">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>318</width>
    <height>323</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>290</y>
     <width>166</width>
     <height>23</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="complex">
   <property name="geometry">
    <rect>
     <x>298</x>
     <y>318</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QCheckBox" name="sh_pass">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>240</y>
     <width>118</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Show Password</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>11</y>
     <width>291</width>
     <height>201</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <property name="labelAlignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <item row="0" column="0">
     <widget class="QLabel" name="label_4">
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="text">
       <string>Username</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="InUser"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Domain</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="InDomain"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>Description</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="InDesc">
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
      <property name="placeholderText">
       <string>Optional</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Password</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="InPass">
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Repeat Password</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="InPass2">
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Entropy</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QProgressBar" name="score">
      <property name="value">
       <number>24</number>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>Score</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QLabel" name="entropy">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="match_pass">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>260</y>
     <width>181</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="complex2">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>220</y>
     <width>179</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>layoutWidget</zorder>
  <zorder>buttonBox</zorder>
  <zorder>complex</zorder>
  <zorder>sh_pass</zorder>
  <zorder>match_pass</zorder>
  <zorder>complex2</zorder>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>AddEntry</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
